src/utils/read-pacts.ts

```diff
@@ -8,13 +8,26 @@ export function readPacts(dir: string): Pact[] {
         throw new Error(`Pact directory ${dir} is empty.`);
     }
 
-    return fileNames.map((fileName) => {
+    const pacts = fileNames.map((fileName) => {
         const filePath = path.join(dir, fileName);
-        return JSON.parse(fs.readFileSync(filePath, 'utf8'));
+        return JSON.parse(fs.readFileSync(filePath, 'utf8')) as Pact;
     });
+
+    const pactWithEmptyInteractions = pacts.find(
```

Comment:
Is it not better to find all invalid pacts with empty interactions and then throw their names all gathered in the error message?

```
const pactsWithEmptyInteractions = pacts.filter(() => {})

if (pactsWithEmptyInteractions.length) {
     const errorMessage = pactsWithEmptyInteractions.reduce((errorMessage, pact) => {}, '');
     throw new Error(errorMessage);
}
```

It would be nicer for developer when he will see that something is wrong with generating all pacts, not only one.